Backstories/Backstories.xml

```diff
@@ -2631,11 +2631,11 @@
   
   <IceworldSurvivor75>
     <!-- EN: iceworld survivor -->
-    <title>TODO</title>
+    <title>Uomo glaciale</title>
     <!-- EN: iceborn -->
-    <titleShort>TODO</titleShort>
+    <titleShort>Uomo glaciale</titleShort>
     <!-- EN: [PAWN_nameDef] was born on an iceworld. Survival depended on staying together and building with nothing.\n\n[PAWN_pronoun] got used to the cold. [PAWN_pronoun] never got used to dealing with those strange green things called plants. -->
-    <desc>TODO</desc>
+    <desc>Nato tra i ghiacci, ha imparato a sopravvivere grazie al lavoro di gruppo e a costruire col nulla.</desc>

```

Comment:
Devi tradurre anche questo:
\n\n[PAWN_pronoun] got used to the cold. [PAWN_pronoun] never got used to dealing with those strange green things called plants. Rigo 2638.